login-workflow/src/new-architecture/screens/AccountDetailsScreen/types.ts

```diff
@@ -0,0 +1,16 @@
+import { TextFieldProps } from '@mui/material/TextField';
+import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.types';
+
+export type AccountDetailsScreenProps = WorkflowCardProps & {
+    // used to test the input for valid formatting
+    firstNameLabel?: string;
+    initialFirstName?: string;
+    firstNameValidator?: (firstName: string) => boolean | string;
+    firstNameTextFieldProps?: Omit<TextFieldProps, 'label' | 'inputRef' | 'value' | 'onChange' | 'onKeyPress'>;
```

Comment:
If we're going to follow this pattern for all the component input fields we should move this type to a more global space. And then we would have something like this... (do not directly commit this)

```
// components/types.ts
export type MinifiedTextFieldProps: Omit<TextFieldProps, 'label' | 'inputRef' | 'value' | 'onChange' | 'onKeyPress'>;
```
```
//  AccountDetailsScreen/types.ts
import { MinifiedTextFieldProps } from '../../components/types;
firstNameTextFieldProps?: MinifiedTextFieldProps
```